core/micro-services/workflow-core/src/main/protobuf/edu/uci/ics/amber/workflowruntimestate.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package edu.uci.ics.amber;
```

Comment:
put it under core

Reply:
nvm. other protobuf are not under core right now. we should change that later together.